src/services/qbit.ts

```diff
@@ -125,12 +125,10 @@ export class QBitApi {
   }
 
   setTorrentName(hash, name) {
-    const params = {
-      hash,
-      name
-    }
-
-    return this.axios.get('/torrents/rename', { params })
+    return this.execute('post', '/torrents/rename', {
+      hash: hash,
```

Comment:
Just nitpicking here but, can be rewritten as just `hash`

Reply:
done!